### PR TITLE
feat: add Trade Tracker

### PR DESCRIPTION
**Pull request by thauanvargas**

### CODE REVIEW

**Comment by WiredSpast** [comment]
Clearly state which data is being shared with the API's in the description

File: store/extensions/Trade Tracker/extension.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "name": "GoEarth",
+    "version": "0.4.0"
+  },
+
+  "language": "Go",
+  "commands": {
+    "default": ["cmd", "/C", "arcade-tracker.exe", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"],
+    "windows": ["cmd", "/C", "arcade-tracker-win.exe", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"],
+    "mac": ["chmod", "+x", "arcade-tracker-mac", "&&", "./arcade-tracker-mac", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"],
+    "linux": ["chmod", "+x", "arcade-tracker-linux", "&&", "./arcade-tracker-linux", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"]
```
Comment (b7c) [inline store/extensions/Trade Tracker/extension.json]: due to #260 I recommend using this format for the Linux and Mac commands
```json
["/bin/sh", "-c", "chmod +x arcade-tracker-linux && ./arcade-tracker-linux -p \"$1\" -c \"$2\" -f \"$3\"", "--", "{port}", "{cookie}", "{filename}"]
````

Comment (thauanvargas) [reply store/extensions/Trade Tracker/extension.json]: I don't have a mac to build with xcode... need some help doing this are you able to?

Comment (b7c) [reply store/extensions/Trade Tracker/extension.json]: the Go compiler can cross compile for Mac easily as long as your project doesn't use CGO, see https://github.com/xabbo/goearth/wiki/Building

Comment (thauanvargas) [reply store/extensions/Trade Tracker/extension.json]: done, thank you

Comment (b7c) [reply store/extensions/Trade Tracker/extension.json]: your mac command is using the linux binary btw

**Pull request merged**